### PR TITLE
s2u-46 5.3.1: Update Messages_ca.properties

### PR DESCRIPTION
**Pull request by DavidVegaSa**

### CODE REVIEW

**Review by mpellicer — CHANGES_REQUESTED** [review]
In general, I see the English translation at the end of the string,remove it please

File: microsoft-integration/admin-tool/src/main/resources/Messages_ca.properties [hunk 1]
```diff
@@ -1,183 +1,183 @@
...
-menu_new.title= Create a new site synchronization
-menu_edit = Edit
-menu_edit.title = Edit a site synchronization
-menu_config = Config
-menu_config.title = Edit microsoft configuration
-menu_credentials = Credentials
-menu_credentials.title = Microsoft Credentials
+menu_main = Administrador
+menu_new = Nou
+menu_new.title= Crea una nova sincronitzaci\00F3 d\0027un espai
```
Comment (mpellicer) [inline microsoft-integration/admin-tool/src/main/resources/Messages_ca.properties:7]: Debe ser \u00f3, corrige el resto de entidades

**Comment by mpellicer** [comment]
@DavidVegaSa Good job so far, do you think the CA translation is ready?

**Comment by DavidVegaSa** [comment]
> @DavidVegaSa Good job so far, do you think the CA translation is ready?

now yes

**Pull request merged**